core/contracts/financial_templates/FeePayer.sol

```diff
@@ -70,6 +70,25 @@ contract FeePayer is Testable {
         return regularFee.add(latePenalty);
     }
 
+    /**
+     * @notice Pays UMA DVM final fees to the Store contract. This is a flat fee charged for each price request.
+     * @return the amount of collateral that was paid to the Store.
+     */
+    function _payFinalFees(address payer) internal returns (FixedPoint.Unsigned memory totalPaid) {
+        StoreInterface store = _getStore();
+        FixedPoint.Unsigned memory finalFee = store.computeFinalFee(address(collateralCurrency));
+
+        if (finalFee.isGreaterThan(0)) {
+            if (payer != address(this)) {
+                require(collateralCurrency.transferFrom(payer, address(this), finalFee.rawValue));
```

Comment:
nit: prefer to use the open zeppelin safe ERC20 methods:
```
collateralCurrency.safeTransferFrom(payer, address(this), finalFee.rawValue);
```

Reply:
agreed good catch